docs/quickstarts/expo.mdx

```diff
@@ -31,103 +55,201 @@ Once you have an Expo application ready, you need to install Clerk's Expo SDK.
   ```
 </CodeBlockTabs>
 
-### Set environment variables
+<Callout type="warning">
+  Currently the `@clerk/expo` SDK only supports mobile app output. You can upvote the addition of Clerk supporting web output [here](https://feedback.clerk.com/roadmap/fb56a8db-763a-4e0d-a4b2-f8a453be195c).
+</Callout>
+
+### Set your environment variables
+
+<SignedIn>
+
+Add the following keys to your `.env` file. These keys can always be retrieved from your project's [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page in the Clerk Dashboard.
+
+</SignedIn>
 
-Below is an example of an `app.config.js` file. To get your keys, go to the API Keys page on the [Clerk dashboard](https://dashboard.clerk.com).
+<SignedOut>
+
+1. Navigate to the Clerk Dashboard.
+2. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+3. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+4. Paste your keys into your `.env` file.
+
+The final result should resemble the following:
+
+</SignedOut>
+
+```sh filename=".env"
+EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+```
+
+Your `app.config.js` might look like this:
```

Comment:
@alexisintech I'm going to remove this section, it was leftover from Shaquil removing an old way of adding env variables to app.config -- which isn't needed and now this is just an incomplete config file reference